mlflow/utils/search_utils.py

```diff
@@ -366,7 +367,7 @@ def _validate_order_by_and_generate_token(cls, order_by):
     def _parse_order_by_string(cls, order_by):
         token_value = cls._validate_order_by_and_generate_token(order_by)
         is_ascending = True
-        tokens = token_value.split()
```

Comment:
One quick question @jdlesage : `filter_string` supports quote parsing. Was there a reason the code was not reused or consolidated? Seems like quote handling operations are different handled for these 2 arguments.

Reply:
There are no specific reasons. I didn't know `fitler_string` manages correctly the quote parsing (I didn't test it). I agree it could be a nice improvement to factorize the 2 ways to manage quotes.